docs/content/reference/cli/authelia/authelia_storage_bans_ip_list.md

```
@@ -0,0 +1,70 @@
+
+Lists ip bans.
+
+This subcommand allows listing ip bans from the regulation system.
+
+```
+authelia storage bans ip list [flags]
+```
+
+### Examples
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)